plugins/additional/additional.cpp

```diff
@@ -1806,24 +1806,24 @@ void ComponentEvtHandler::OnMarginClick( wxStyledTextEvent& event )
 class DataViewModel : public wxDataViewModel
 {
 public:
-	unsigned int GetChildren(const wxDataViewItem& item, wxDataViewItemArray& children) const {
+	unsigned int GetChildren(const wxDataViewItem& WXUNUSED(item), wxDataViewItemArray& WXUNUSED(children)) const {
```

Comment:
Are there any advantages of `WXUNUSED` over just commenting out the name with `/* */`?

Reply:
Portability. I don't know how much this is an issue today, but i remember having read that some compilers don't support this and require other means to ignore unused parameters. Thats why WXUNUSED, boost::ignore_unused() and the like exist, so i used the wxWidgets style here.

Reply:
Okay, I think that isn't an issue today anymore.

Reply:
It is/was wxWidgets style to suppress these warnings, i would keep them so its easier in the future to replace them with [[maybe_unused]]

Reply:
`[[maybe_unused]]` is even longer and "maybe" just seems like "disable the warning" to me. But it's a good warning because it can catch some problems (e.g. mistyping) or unnecessary parameters.

In this case, I think it's actually best to just remove the parameter name, as "item" doesn't add any extra information anyway.

I've fixed the warning with `/* */` or by removing the name in 0985941f5e4dac3c1a46ae029cbb66cd257d1b97.